src/components/BulkEditList/BulkEditListFilters/BulkEditListFilters.js

```diff
@@ -45,10 +45,14 @@ export const BulkEditListFilters = ({
     isUserRadioDisabled,
     hasInAppEditPerms,
     isSelectIdentifiersDisabled,
+    hasLogViewPerms,
   } = useBulkPermissions();
   const search = new URLSearchParams(location.search);
 
   const { criteria, capabilities, recordIdentifier, queryText } = filters;
+  const isQuery = criteria === CRITERIES.QUERY;
+  const isLogs = criteria === CRITERIES.LOGS;
+  const isIdentifier = criteria === CRITERIES.IDENTIFIER;
```

Comment:
I don't recognize the word `criteries`. Is it maybe supposed to be `criteria`? I realize that isn't new code in this PR, but it would be nice to get it corrected.

Reply:
`criteria` is plural, `criterion` is singular. I think we want `criteria` here. Or maybe we should skip the Latin holdover and just use a straight-up English word like `options`? 😆